cbt.go

```diff
@@ -474,11 +474,12 @@ var commands = []struct {
 		Name: "createtable",
 		Desc: "Create a table",
 		do:   doCreateTable,
-		Usage: "cbt createtable <table-id> [families=<family>:<gcpolicy-expression>,...]\n" +
+		Usage: "cbt createtable <table-id> [families=<family>:<gcpolicy-expression>:<type-expression>,...]\n" +
 			"   [splits=<split-row-key-1>,<split-row-key-2>,...]\n" +
-			"  families     Column families and their associated garbage collection (gc) policies.\n" +
+			"  families     Column families and their associated garbage collection (gc) policies and types.\n" +
 			"               Put gc policies in quotes when they include shell operators && and ||. For gcpolicy,\n" +
 			"               see \"setgcpolicy\".\n" +
+			"               Currently only the type \"intsum\" is supported.\n" +
```

Comment:
isn't there also byte type?

Reply:
not publicly supported (you can't create a column of type Bytes) so I didn't mention it here or add it to the parser.